examples/scala/README.md

```diff
@@ -0,0 +1,14 @@
+# Run
```

Comment:
Can you please add a note to the readme files that the examples are running version X, and to change it they need to change Y in file Z?